web/src/test/java/org/springframework/security/web/authentication/www/BasicAuthenticationFilterTests.java

```diff
@@ -424,4 +424,20 @@ public void doFilterWhenTokenAndFilterCharsetDoNotMatchThenUnauthorized() throws
 		assertThat(SecurityContextHolder.getContext().getAuthentication()).isNull();
 	}
 
+	@Test
+	public void requestWhenEmptyBasicAuthorizationHeaderTokenThenIgnored() throws Exception {
```

Comment:
Please update the test name, since the header is not ignored, but rather returns unauthorized.
```suggestion
	public void requestWhenEmptyBasicAuthorizationHeaderTokenThenUnauthorized() throws Exception {
```